src/components/MAPI/workernodes/WorkerNodesCreateNodePool.tsx

```diff
@@ -299,6 +309,16 @@ const WorkerNodesCreateNodePool: React.FC<IWorkerNodesCreateNodePoolProps> = ({
                 cluster={cluster}
                 margin={{ top: 'small' }}
               />
+
+              {supportsSpotInstances && (
```

Comment:
We only display the spot instances configuration components for clusters that support it (release version >= `14.1.0`)